OPS.md

```diff
@@ -0,0 +1,56 @@
+# Vox Pupuli DevOps
+
+This is an overview of our infrastructure
+
+## System rules
+
+* We manage systems via puppet apply and [our controlrepo](https://github.com/voxpupuli/controlrepo)
+* they are all under the `voxpupu.li` domain
+
+## Systems
+
+### ci01.voxpupu.li
+
+* Dedicated X86 server sponsored by [macarne.com](https://macarne.com/).
+* It hosts different CI runners
+
+### ci02.voxpupu.li
+
+* ARM64 VM at [Hetzner](https://hetzner.de/), sponsored by simonhoenscheid
+
+### CERN CI runners
+
+* Different VMs managed by CERN
+* Currently configured as runners for the whole [voxpupuli GitHub org](https://github.com/organizations/voxpupuli/settings/actions/runner-groups/4)
+* contact person is straylen
+* For incidents contact `service-desk@cern.ch`
+
+## Software as a service
+
+Admin access to the services is handled via shared accounts in `puppet_credentials` or via personal accounts.
+Those are explained in the governance.md doc in the PMC section.
+
+### public mailinglist
+
+[groups.io](https://groups.io/g/voxpupuli) runs the `voxpupuli@groups.io` mailinglist.
+
+### PMC alias
+
+`pmc@voxpupuli.org` is configured at [mailgun](https://app.mailgun.com/mg/sending/mailing-lists/pmc%40voxpupuli.org?listMembers=%7B%22limit%22%3A10%2C%22searchAddress%22%3A%22%22%2C%22skip%22%3A0%7D).
+
+### internal PMC email addresses
+
+We've two internal email addresses.
+They shouldn't be used publicly.
+They are listed in the `puppet_credentials`.
+
+### DNS
+
+Our domain registrar is [INWX](https://www.inwx.de/de).
```

Comment:
Perhaps nice to include if we pay for this. If so, Foreman has its domain sponsored by Gandi and we can see if they're willing to sponsor another open source project.

Reply:
updated it